packages/pn-personafisica-webapp/cypress/e2e/Delegate.cy.ts

```diff
@@ -13,51 +18,135 @@ describe('Delegation', () => {
+        statusCode: 200
+      }).as('rejectDelegation');
+      cy.get('.css-pj2eij-MuiGrid-root > [data-testid="dialogAction"]').click();
+      cy.wait('@rejectDelegation');
+      cy.get('.css-19kzrtu > :nth-child(3)').contains(/Giuseppe Maria Garibaldi/).should('not.exist');
+    });
+
+    it('Should accept delegator\s request', () => {
+      cy.get('[data-testid="menu-list"] > :nth-child(4) > .MuiBox-root').contains(/1/).should('exist');
+      cy.visit(DELEGHE);
+      cy.wait('@getDelegators');
+      cy.get('.MuiTableCell-alignCenter > .MuiButtonBase-root').click();
+      const verificationCode = '25622';
+      verificationCode.split('').forEach((code, i) =>
+        cy.get(`:nth-child(${i + 1}) > .MuiInputBase-root > #outlined-basic`).type(code)
+      );
+      cy.intercept(`${DELEGATIONS_BY_DELEGATE()}`, {
+        fixture: 'delegations/mandates-by-delegate-after-activation',
+      }).as('getDelegatesAfterActivation');
+      cy.intercept('PATCH',`${ACCEPT_DELEGATION('af02d543-c67e-4c64-8259-4f7ac12249fd')}`, {
+        statusCode: 204
+      }).as('acceptDelegation');
+      cy.get('.MuiDialogActions-root > .MuiButton-contained').click();
+      cy.wait('@getDelegatesAfterActivation');
+      cy.get(':nth-child(2) > .MuiTableCell-alignCenter > .MuiChip-root > .MuiChip-label').contains(/Attiva/).should('exist');
+    });
+  });
+
+  describe('As delegator', () => {
+    it('Should create new delegate', () => {
+      cy.visit(DELEGHE);
+      cy.get('.css-1teipz8-MuiStack-root > .MuiBox-root > .MuiButtonBase-root').click();
+      const formDelegator = {
+        name: 'Cristoforo',
+        surname: 'Colombo',
+        fiscalCode: 'CLMCST42R12D969Z',
+      }
+      cy.get('#nome').type(formDelegator.name);
+      cy.get('#cognome').type(formDelegator.surname);
+      cy.get('#codiceFiscale').type(formDelegator.fiscalCode);
+      cy.intercept('POST', `${CREATE_DELEGATION()}`, {
+        statusCode: 201,
+        fixture: 'delegations/mandate'
+      }).as('createDelegation');
+      cy.get('[data-testid="createButton"]').click();
+      cy.wait('@createDelegation');
+      cy.get('.MuiTypography-h4').contains(/La tua richiesta di delega è stata creata con successo/).should('exist');
+      cy.intercept(`${DELEGATIONS_BY_DELEGATOR()}`, {
+        fixture: 'delegations/mandates-by-delegator-after-new-delegation',
+      }).as('getDelegatorsAfterNewDelegation');
+      cy.visit(DELEGHE);
+      cy.wait('@getDelegatorsAfterNewDelegation');
+      cy.get(':nth-child(2) > .css-1uen9zh-MuiTableCell-root > .MuiTypography-root').contains(/Cristoforo Colombo/).should('exist');
+      cy.get(':nth-child(1) > .MuiTableCell-alignCenter > .MuiChip-root > .MuiChip-label').contains(/In attesa di conferma/).should('exist');
+    });
+
+    it('Shoud revoke a delegate', () => {
+      cy.visit(DELEGHE);
+      cy.wait('@getDelegators');
+      cy.intercept('PATCH', `${REOVKE_DELEGATION('6c969e5d-b3a0-4c11-a82a-3b8360d1436c')}`, {
```

Comment:
There's a typo with this route. I suggest reporting it in order to open an issue.